document/core/conf.py

```diff
@@ -477,6 +486,8 @@
 
 # Macros
 rst_prolog = """
+.. |repolink| replace:: https://github.com/webassembly/""" + repo + """/issues/
```

Comment:
Maybe `|issuelink|` instead, since this points to the issues rather than the main repo page?

Reply:
Done. I originally had defined it as the actual repolink, but Sphinx substitutions are too braindead to express a link constructed by extension.